src/components/ChataTable/ChataTable.js

```diff
@@ -672,6 +697,17 @@ export default class ChataTable extends React.Component {
   //   this.ref?.tabulator?.clearHeaderFilter()
```

Comment:
Remove?

Reply:
See comment above this line. Just want to keep this for future feature of "clear all filters" button. It needs a little more work and not a huge priority for now